### PR TITLE
docs(search-forms): update documentation - INNO-769

### PR DESCRIPTION
**Pull request by yhuard**
# PR description

Just a documentation update

### CODE REVIEW

File: framework/components/ecl-search-forms/README.md [hunk 1]
```diff
@@ -2,11 +2,19 @@
...
+#### Internal search
+
+This search box allows you to search only within the pages of the current website.
 
 ## When to use this component
 
-- It is located on the the site header.
-- If a site/section search is required, it can be implemented below the page header.
+- the corporate search is located on the the site header
+- if a internal search is required, it can be implemented below the page header
```
Comment (kalinchernev) [inline framework/components/ecl-search-forms/README.md]: `an internal` search please, even it's different than original, let's correct it.

File: framework/components/ecl-search-forms/README.md [hunk 2]
```diff
@@ -2,11 +2,19 @@
...
 
-The search box allow user to search within all EC websites. The search box allows user to search in all European Commission websites through the EC search engine.
+A "Search" functionality helps users to locate content quicker by using specific words or phrases without having to navigate through the entire website.
 
-The search results are displayed in a separate page.
+### Types of search
+
+#### Corporate search
+
+The search box allows the user to search within all EC websites through the EC search engine. The search results are displayed in a separate page.
```
Comment (kalinchernev) [inline framework/components/ecl-search-forms/README.md]: Please take also the rest of the information, even if it's not super related to the component ...

**Review by kalinchernev — CHANGES_REQUESTED** [review]
Small missing part.

File: framework/components/ecl-search-forms/README.md (hunk 2 above)
Comment (yhuard) [inline framework/components/ecl-search-forms/README.md]: This information was Drupal-specific, I've dropped it. @degliwe has commented the spreadsheet

**Pull request merged**